pss/forwarding_test.go

```diff
@@ -128,7 +129,7 @@ func TestForwardBasic(t *testing.T) {
 	}
 	testCases = append(testCases, c)
 
-	// test with partial addresses
+	//test with partial addresses
```

Comment:
Why removing the whitespace?

Reply:
Commenting and uncommenting the code resulted in this. I added back the whitespace.